setup.ocm.py

```diff
@@ -0,0 +1,40 @@
+import setuptools
+import os
+
+own_dir = os.path.abspath(os.path.dirname(__file__))
+
+
+def requirements():
+    with open(os.path.join(own_dir, 'requirements.ocm.txt')) as f:
+        for line in f.readlines():
+            line = line.strip()
+            if not line or line.startswith('#'):
+                continue
+
+            yield line
+
+
+def modules():
+    return [
+    ]
+
+
+def version():
+    with open(os.path.join(own_dir, 'ocm', 'VERSION')) as f:
+        return f.read().strip()
+
+
+setuptools.setup(
+    name='ocm',
+    version=version(),
+    description='Open-Component-Model (OCM) language bindings',
+    python_requires='>=3.11',
+    py_modules=modules(),
+    packages=['ocm', 'gci'],
```

Comment:
if we include the whole `gci` module, we will also have to install `oci` requirements
see https://github.com/gardener/cc-utils/blob/84731c906ceab2026c3ff9b95f1969b10989a857/gci/oci.py#L11

Reply:
that is indeed correct. I think I will, however, remove reference to `gci`, rather than adding those dependencies, for now.